pyfr/readers/base.py

```diff
@@ -22,3 +26,209 @@ def to_pyfrm(self):
         mesh['mesh_uuid'] = np.array(str(uuid.uuid4()), dtype='S')
 
         return mesh
+
+
+class BaseMeshConverter(object):
```

Comment:
The design of this class is a bit fragile.  It requires anyone inheriting from it to set a bunch of internal variables.  Could do with a minor rework to abstract this.  (Perhaps the constructor can take element and face data as parameters to avoid sub-classes have to set various properties themselves.